MANIFEST.in

```diff
@@ -1,3 +1,3 @@
-include *.rst
-recursive-include ihatemoney *.rst *.py *.yaml *.po *.mo *.html *.css *.js *.eot *.svg *.woff *.txt *.png *.webp *.ini *.cfg *.j2 *.jpg *.gif *.ico *.xml
-include LICENSE CONTRIBUTORS CHANGELOG.rst
+include *.md
+recursive-include ihatemoney *.py *.yaml *.po *.mo *.html *.css *.js *.eot *.svg *.woff *.txt *.png *.webp *.ini *.cfg *.j2 *.jpg *.gif *.ico *.xml
```

Comment:
Why not adding `*.md` it also in the recursive-include ? (wouldn't the doc be excluded by this?)

Reply:
We have no markdown file in ihatemoney/, do you think it's useful?

Reply:
I was mainly thinking about the documentation. It would be nice to have it included in the release :-)